src/app/prihlasit/page.tsx

```diff
@@ -82,6 +95,12 @@ const LoginPage: NextPage = () => {
             errors={errors}
           />
           <PasswordInput register={register} errors={errors} />
+          {errorMessage && (
+            <AriaLiveErrorMessage
```

Comment:
Funfact AriaLive must always be mounted. I do believe it should be something high in the chain, like a root. You cannot mount it as needed. 

I was not part of the team that was fixing it on Monster, but that is what we had to do